watertap/examples/flowsheets/lsrro/tests/test_lsrro.py

```diff
@@ -193,32 +193,37 @@ def test_set_operating_conditions(self, model):
         assert degrees_of_freedom(model) == 0
         self._test_no_badly_scaled_vars(model)
 
+    @pytest.mark.requires_idaes_solver
     @pytest.mark.component
     def test_initialize(self, model, initialization_data):
         initialize(model)
         for var, val in initialization_data.items():
             assert pyo.value(var) == pytest.approx(val, rel=1e-5)
         self._test_no_badly_scaled_vars(model)
 
+    @pytest.mark.requires_idaes_solver
     @pytest.mark.component
     def test_simulation(self, model, simulation_data):
         solve(model)
         for var, val in simulation_data.items():
             assert pyo.value(var) == pytest.approx(val, rel=1e-5)
         self._test_no_badly_scaled_vars(model)
 
+    @pytest.mark.requires_idaes_solver
     @pytest.mark.component
     def test_display_system(self, model, capsys):
         display_system(model)
         captured = capsys.readouterr()
         assert captured.out == self.display_system
 
+    @pytest.mark.requires_idaes_solver
     @pytest.mark.component
     def test_display_design(self, model, capsys):
         display_design(model)
         captured = capsys.readouterr()
         assert captured.out == self.display_design
 
+    @pytest.mark.requires_idaes_solver
```

Comment:
Does this mean we shouldn't use ipopt-watertap when working with LSRRO?

Reply:
No, the difference is insignificant, but doesn't seem worth resolving. This also only affects those without an IDAES provided Ipopt. Note that `ipopt-watertap` is just a wrapper around *whatever* Ipopt the user has.

On my mac without these xfails:
```
========================================================= short test summary info =========================================================
FAILED tests/test_lsrro.py::TestLSRRO_2Stage::test_initialize - assert 0.2985162459059866 == 0.298029 ± 3.0e-06
FAILED tests/test_lsrro.py::TestLSRRO_3Stage::test_initialize - assert 0.35344241623557743 == 0.351684 ± 3.5e-06
================================================= 2 failed, 25 passed in 60.02s (0:01:00) =================================================
```

But, reconsidering it, I think the better answer is just relaxing the relative tolerance, especially given this is just initialization.